crates/rpc/src/lib.rs

```diff
@@ -159,6 +161,17 @@ pub mod test_utils {
         let class1_hash = ClassHash(felt_bytes!(b"class 1 hash"));
         let class2_hash = ClassHash(felt_bytes!(b"class 2 hash (sierra)"));
 
+        let state_diff0 = StateDiff::default()
+            .add_deployed_contract(contract0_addr, class0_hash)
+            .add_nonce_update(contract0_addr, ContractNonce(felt!("0x1")));
+
+        let state_diff1 = StateDiff::default().add_deployed_contract(contract1_addr, class1_hash);
+
+        let state_diff2 = StateDiff::default()
+            .add_deployed_contract(contract2_addr, class2_hash)
+            .add_nonce_update(contract1_addr, ContractNonce(felt!("0x10")))
+            .add_nonce_update(contract2_addr, ContractNonce(felt!("0xfeed")));
+
```

Comment:
I'll expand more on this sort of building (and using this as the foundation for the rest of setup).